zookeeper/conf/zoo.cfg

```diff
@@ -47,7 +47,7 @@ clientPortAddress={{ bind_address }}
 
 {%- for garbage_string in zookeepers %}
   {%- set myid, mynodename = garbage_string.split('+') %}
-server.{{ myid }}={{ mynodename }}:2888:3888
+server.{{ myid }}={{ mynodename }}:{{ quorum_port}}:{{election_port}}
```

Comment:
Minor nitpick, but can we fix the variable spacing on `{{ quorum_port}}:{{election_port}}`?